commands/hugo.go

```diff
@@ -578,9 +580,16 @@ func (c *commandeer) copyStatic() error {
 	syncer.DestFs = c.Fs.Destination
 	// Now that we are using a unionFs for the static directories
 	// We can effectively clean the publishDir on initial sync
-	syncer.Delete = c.Cfg.GetBool("cleanDestinationDir")
+	syncer.Delete = c.Cfg.GetBool("cleanDestinationDir") || c.Cfg.GetBool("cleanDestinationDirKeepDotFolders")
```

Comment:
No problem, I'm not sure where to update the documentation, do I just update `commands/hugo.go`?

ln:215:
`cmd.Flags().Bool("cleanDestinationDir", false, "Remove files from destination not found in static directories. Folders beginning with "." are ignored")`